package.json

```diff
@@ -34,7 +35,8 @@
     "eslint-config-xo-space": "^0.15.0",
     "eslint-plugin-json": "^1.2.0",
     "eslint-plugin-mocha": "^4.12.1",
-    "mocha": "^3.5.3"
+    "mocha": "^3.5.3",
+    "sinon": "^7.4.2"
```

Comment:
We don't use the `^` in our package.json files.  Can you remove all the `^` characters for both the new and old pacakges?   Also, now that you use the `@elastic.io/component-commons-library`, then `@elastic.io/jsonata-moment` is likely redundant and can be removed as well.